languages/elixir/exercises/concept/processes/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+You are writing an embedded system for a Take-A-Number machine. It is a very simple model. It can give out consecutive numbers and report what was the last number given out.
+
+## 1. Start the machine
+
+Implement the `start/0` function. It should spawn a new process that has an initial state of `0` and is ready to receive messages. It should return the process' PID.
+
+```elixir
+TakeANumber.start()
+# => #PID<0.138.0>
```

Comment:
Is it worthwhile to mention that the pid may be different when they run it as it is generated by the runtime?

Reply:
👍 [`801fb7e` (#1654)](https://github.com/exercism/v3/pull/1654/commits/801fb7ece08169bab4f61972eb58a0be06ebf43b)